### DISCUSSION TITLE
How to update LHC from SSH avoiding updating from SFTP that require long time?

### DISCUSSION THREAD

**Discussion by PeopleInside** [LiveHelperChat/livehelperchat#2144] [Q&A]
Hi,
I was looking the doc page for the upgrading process https://doc.livehelperchat.com/docs/upgrading

Every time a new version of LHC is available I download the zip file, I unzip and I upload only the suggested folder but this require long time, sometimes more then 30 minutes as SFTP is not fast.

You are using `git pull`?
Can I ask how it works?

To update with this method I have just to going in the live helper chat folder then run `git pull` as written or there is a full different command?
Before lunch this command database should be updated in the user interface or the git pull command will update also the database?

Another question:
For keep extensions updated (for example Telegram) I need send other SSH command or git pull will also update extensions?
With command I should use for keep extensions and his database updated?

Thank you very much.
---
I see some other instructions here: https://livehelperchat.com/how-to-automate-live-helper-chat-updates-338a.html
`git pull origin master` is the exact command that should be run in the live helper chat root folder?

**Comment by mysubcult** [comment]
cd /var/www/html/livehelperchat && git pull origin master
cd /var/www/html/public_html && php cron.php -s site_admin -c cron/util/update_database

**Reply by PeopleInside** [reply]
Thank you for the reply!
I tried this method but I get an error with `git pull origin master` so I returned to update all time with SFTPS that is very slow 😁.
Maybe I get error because I need run a command to initialize the project as I installed LHC by SFTPS.
Maybe the command `git init` is necessary but on my test I was not able to made it work correctly.

Maybe is too hard for me 😜😊

**Reply by mysubcult** [reply]
> Thank you for the reply! I tried this method but I get an error with `git pull origin master` so I returned to update all time with SFTPS that is very slow 😁. Maybe I get error because I need run a command to initialize the project as I installed LHC by SFTPS. Maybe the command `git init` is necessary but on my test I was not able to made it work correctly.
> 
> Maybe is too hard for me 😜😊

To help you, you need to understand what your mistake is.
